osquery/filesystem/windows/fileops.cpp

```diff
@@ -154,6 +154,142 @@ Status windowsGetVersionInfo(const std::string& path,
   return Status::success();
 }
 
+typedef struct LANGANDCODEPAGE {
+  WORD wLanguage;
+  WORD wCodePage;
+} langandcodepage_t;
+
+// retrieve the list of languages and code pages from version information
+// ressource
+BOOL getLanguagesAndCodepages(
```

Comment:
I think the PR looks good, do you mind changing this and `getOriginalFilenameForCodepage` to return a `Status` to be consistent with the rest of the APIs here?

Reply:
Of course, I am going to change this. Thanks!

Reply:
Of course, I am going to change this. Thanks!